src/sentry/ingest/consumer/processors.py

```diff
@@ -149,10 +153,22 @@ def process_event(
         if reprocess_only_stuck_events and not event_processing_store.exists(data):
             return
 
+        is_reprocessed = event_processing_store.exists(data)
```

Comment:
please put this in an else block of the previous block so you do not read from redis twice (exists).
```
event_exists = event_processing_store.exists(data)
if reprocess_only_stuck_events and not event_exists:
     return
if event_exists:
   ...the rest ...
```

Reply:
oh actually, this can be removed entirely right? since we'll never proceed past line 154 unless we're reprocessing